recipes/gargammel/meta.yaml

```diff
@@ -5,7 +5,7 @@ package:
   version: {{ version }}
 
 build:
-  number: 0
+  number: 2
```

Comment:
not sure wh 2, but it also does not hurt :)

Reply:
Because some dummy didn't merge `bulk` into `master` yet :O. Once I get those builds (and their downstream packages) I'll finally merge `bulk` again.